<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="filterBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="sImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="nImage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pImage">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="selectedImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="filteredImage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;--&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QSlider" name="noiseSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="noiseLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="errLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Errors count: </string>
    </property>
   </widget>
   <widget class="QLabel" name="errPLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Errors percent:</string>
    </property>
   </widget>
   <widget class="QLabel" name="errCountLable">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorPTextLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
